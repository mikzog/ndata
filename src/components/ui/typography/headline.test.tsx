import React from 'react';
import renderer from 'react-test-renderer';
import Headline from './headline';

describe('Headline', () => {
  test('renders without crashing', () => {
    const component = renderer.create(<Headline />);
    const json = component.toJSON();

    expect(json).toMatchSnapshot();
  });

  test('renders with correct tag', () => {
    const component = renderer.create(<Headline tag="h3" />);
    const json = component.toJSON();

    expect(json).toMatchSnapshot();
  });
});
