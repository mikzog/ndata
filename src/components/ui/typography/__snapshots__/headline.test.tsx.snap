// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headline renders with correct tag 1`] = `<h3 />`;

exports[`Headline renders without crashing 1`] = `<h1 />`;
